tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_advertiser.c

```diff
@@ -51,6 +55,58 @@ static void adv_init(void)
 	ASSERT_OK(bt_mesh_adv_enable(), "Mesh adv init failed");
 }
 
+static void allocate_all_net_buf(struct net_buf **buf, size_t num_buf, uint8_t xmit)
+{
+	for (int i = 0; i < num_buf; i++) {
+		*buf = bt_mesh_adv_create(BT_MESH_ADV_DATA, BT_MESH_LOCAL_ADV,
+					  xmit, K_NO_WAIT);
+
+		ASSERT_FALSE(!*buf, "Out of buffers");
+		buf++;
+	}
+}
+
+static void verify_net_buf_overflow(void)
+{
+	struct net_buf *dummy_buf;
+
+	/* Verity Queue overflow */
+	dummy_buf = bt_mesh_adv_create(BT_MESH_ADV_DATA, BT_MESH_LOCAL_ADV,
+				       BT_MESH_TRANSMIT(2, 20), K_NO_WAIT);
+	ASSERT_TRUE(!dummy_buf, "Unexpected extra buffer");
+}
+
+static bool check_retr_delta_time(void)
+{
+	static int cnt;
+	static int64_t timestamp;
+	static int64_t delta;
```

Comment:
Doesn't need to be static. Should be aligned with implementation in https://github.com/zephyrproject-rtos/zephyr/pull/50195